.travis/test.sh

```diff
@@ -55,6 +55,12 @@ then
   UT_FLAGS+=" -K FIXME_py3"
 fi
 
+if python --version 2>&1 | grep -q '^Python 2\.'
+then
+  # Some Features are not available in Python 2. This tests fail. 
+  UT_FLAGS+=" -K ONLY_py3"
+fi
+
```

Comment:
Would it be possible to install `python-can` when running on Python 2 so that we also test its implementation ?

Reply:
I already install and test python-can for the pythoncansocket implementation

Reply:
Sorry, missed it

Reply:
Have you also get it installed on Appveyor ? (windows)

Reply:
windows won't help me that much, because there do not exist virtual can interfaces. So I can not simulate the CAN communication. The same applies for osx. Python-Can only supports "real" hardware interfaces on windows

Reply:
Got it